src/main/java/uk/gov/hmcts/reform/sscs/config/HmcHearingsEventTopicListener.java

```diff
@@ -26,49 +31,53 @@
 
+
+    @Autowired
+    private static HearingsJourneyService hearingsJourneyService;
     @Value("${azure.service-bus.hmc-to-hearings-api.connectionString}")
     private String connectionString;
     @Value("${azure.service-bus.hmc-to-hearings-api.topicName}")
     private String topicName;
     @Value("${azure.service-bus.hmc-to-hearings-api.subscriptionName}")
     private String subscriptionName;
-
     @Value("${azure.service-bus.hmc-to-hearings-api.retryTimeout}")
     private Long retryTimeout;
     @Value("${azure.service-bus.hmc-to-hearings-api.retryDelay}")
     private Long retryDelay;
     @Value("${azure.service-bus.hmc-to-hearings-api.maxRetries}")
     private Integer maxRetries;
-
     @Value("${sscs.serviceCode}")
     private String serviceId;
 
-    @Bean
-    @SuppressWarnings("PMD.CloseResource")
-    public void hmcHearingEventProcessorClient() {
-        ServiceBusProcessorClient processorClient = new ServiceBusClientBuilder()
-            .retryOptions(retryOptions())
-            .connectionString(connectionString)
-            .processor()
-            .topicName(topicName)
-            .subscriptionName(subscriptionName)
-            .processMessage(HmcHearingsEventTopicListener::processMessage)
-            .processError(HmcHearingsEventTopicListener::processError)
-            .buildProcessorClient();
-
-        processorClient.start();
-        log.info("HMC hearing event processor started.");
-    }
-
     public static void processMessage(ServiceBusReceivedMessageContext context) {
         ServiceBusReceivedMessage message = context.getMessage();
         HmcMessage hmcMessage = message.getBody().toObject(HmcMessage.class);
         String hmctsServiceID = hmcMessage.getHmctsServiceID();
 
         if (isMessageRelevantForService(hmcMessage, hmctsServiceID)) {
-            //TODO process all messages: SSCS-10286
-            log.info("Processing hearing ID: {} for case reference: {}%n", hmcMessage.getHearingID(),
-                hmcMessage.getCaseRef());
+            try {
+                log.info("Processing hearing ID: {} for case reference: {}%n", hmcMessage.getHearingID(),
+                         hmcMessage.getCaseRef()
+                );
+
+                hearingsJourneyService.process(hmcMessage);
+
+                log.info("Complete hearing ID: {} for case reference: {}%n", hmcMessage.getHearingID(),
+                         hmcMessage.getCaseRef()
+                );
+
+                context.complete();
+            } catch (GetCaseException | UpdateCaseException exc) {
+                log.error("An exception occurred whilst processing hearing event for "
+                              + "HMCTS Service ID {}, case reference: {}", hmcMessage.getHearingID(),
+                          hmcMessage.getCaseRef(), exc);
+                context.abandon();
+            }
+        } else {
+            log.info("Nothing updated for hearing ID: {} for case reference: {}%n", hmcMessage.getHearingID(),
```

Comment:
I think this should be a debug message wrapped in a if (log.isDebugEnabled()) block. It should also read "Skipping hearing ID {} for service {} (hmctsServiceID)